README.md

```diff
@@ -12,12 +12,43 @@ information about installing the `polkadot` binary and developing on the codebas
 specific guides, like how to be a validator, see the
 [Polkadot Wiki](https://wiki.polkadot.network/docs/en/).
 
-## Building
+## Installation
+
+If you just wish to run a Polkadot node without compiling it yourself, you may
+either run the latest binary from our
+[releases](https://github.com/paritytech/polkadot/releases) page, or install
+Polkadot from one of our package repositories.
+
+### Debian-based (Debian, Ubuntu)
+
+Currently supports Debian 10 (Buster) and Ubuntu 20.04 (Focal), and derivatives.
 
-### Use a Provided Binary
+```
+# Import the security@parity.io GPG key
+curl -fsSL 'https://keys.mailvelope.com/pks/lookup?op=get&search=security%40parity.io&options=mr&exact=on' | apt-key add -
+# Add the Parity repository and update the package index
+echo 'deb https://releases.parity.io/deb buster main' >> /etc/apt/sources.list.d/parity.list
```

Comment:
Can we please suggest putting the key into `/usr/share/keyrings/` and using the `deb [signed-by=<path to key>] URL…` format instead?
There's no real reason to add a system-level apt key for a non-system repo.
(Warning: the key should be _dearmored_ for this to work)

Reply:
Resolved this with the latest commit. *Please* let me know if there's any method of getting the key from a keyserver to pipe to a file (unarmored), rather than importing it into their keyring and exporting it (which isn't the worst thing in the world...). There are only so many lines of the gpg manpage I can read in one day ;)